### PR TITLE
refactor: Improve Python import helpers

### PR DESCRIPTION
**Pull request by andiwand**
after my recent changes hepmc3 test will only run if the environment is sourced. this refactors the import helpers in order to fix that

### CODE REVIEW

File: Examples/Python/tests/helpers/__init__.py [hunk 1]
```diff
@@ -33,19 +33,19 @@
...
 
-edm4hepEnabled = "EDM4HEP_DIR" in os.environ
-if edm4hepEnabled:
-    try:
-        import acts.examples.edm4hep
-    except ImportError:
-        edm4hepEnabled = False
+    hepmc3Enabled = True
+except ImportError:
+    hepmc3Enabled = False
```
Comment (AJPfleger) [inline Examples/Python/tests/helpers/__init__.py:41]: Would it make sense to also change the above `geant4Enabled` (l.8-13) and `dd4hepEnabled` (l.29-34) to this cleaner version? (as also already done with `rootEnabled`)

Comment (paulgessinger) [reply Examples/Python/tests/helpers/__init__.py:41]: The underlying issue is that `edm4hep` and `hepmc3` weren't importing additional modules in the helper python files. Geant4, DD4hep and ROOT are not functional without the runtime environment variables set, so this makes sense to have differently in my opinion.

**Comment by andiwand** [comment]
waiting for https://gitlab.cern.ch/acts/OpenDataDetector/-/merge_requests/34

**Comment by andiwand** [comment]
good to go @paulgessinger ?

**Review by paulgessinger — APPROVED** [review]
LGTM! Let's merge if the CI runs through now without skipped tests.

**Pull request merged**